src/main/java/com/study/stockmanagementstudycase/common/exception/StockAlreadyExistexception.java

```diff
@@ -0,0 +1,20 @@
+package com.study.stockmanagementstudycase.common.exception;
+
+import java.io.Serial;
+
+public class StockAlreadyExistexception extends RuntimeException{
+
+    @Serial
+    private static final long serialVersionUID = -2675930856483765098L;
+
+    private static final String DEFAULT_MESSAGE =
+            "The Stock Name  already exist!";
```

Comment:
Bu çok spesifik bir hata olmuş. Bu hatayı ilgili stoğun zaten var olduğu tüm senaryolarda kullanacağız. Bu sebeple daha genel bir mesaj yazmamız gerekiyor.

Önerim : "The specified stock already exist!"